templates/profile.html

```diff
@@ -0,0 +1,27 @@
+<html>
+
+<head>
+    <!--  
+  <style>
+   body {
+  # Uncomment to add CSS Styling
+        }
+   </style>
+   -->
+</head>
+
+    <!-- Added An HTML Page So User Can Visit A Profile Page Before Main App Page.
+        Clicking A Button Will Route The User To View Main Page -->
+    
+    <!-- This Page Will Be Further Designed To Have Functionality So The User 
+         Can Edit Their User Profile And The Data They Enter Will Be Saved, Updated,  
+         And Displayed -->
+
+<body>
+    <h1>Profile Page</h1>
+    <a href="/index">
+        <button>View Main Page</button>
```

Comment:
I think that  buttons inside anchor elements is nonstandard. In the future, we should add a class to the `a` element and style it as a button instead (`<a class='btn' href='/index'>View Main Page</a>`).